src/dialog/dialog.component.ts

```diff
@@ -63,13 +64,15 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 	/**
 	 * `Subscription` used to update placement in the event of a window resize.
 	 */
-	protected resizeSubscription: Subscription;
+	protected resizeSubscription: Subscription = new Subscription();
 	/**
 	 * Subscription to all the scrollable parents `scroll` event
 	 */
 	// add a new subscription temporarily so that contexts (such as tests)
 	// that don't run ngAfterViewInit have something to unsubscribe in ngOnDestroy
 	protected scrollSubscription: Subscription = new Subscription();
```

Comment:
remove `: Subscription`
```suggestion
	protected scrollSubscription = new Subscription();
```